### DISCUSSION TITLE
simple pdb parsing

### DISCUSSION THREAD

**Discussion by lobnaOmar20** [project-gemmi/gemmi#276] [Q&A]
could anybody please tell me about the feature in Gemmi where I can parse pdb and get the last step in my REMARK ?? 
I have a REMARK of 57 steps where I want only to read the 57th step and write it to another pdb file. 

I looked thoroughly in the documentation but only a few things are written, none of them helpful to my case. 

I think otherwise I will resort to the ordinary way of parsing files like for example txt files but I really want to use Gemmi.

**Comment by wojdyr** [comment]
The REMARK records are stored in `Structure::raw_remarks`.
I don't know how you store your steps in REMARKs. If you just want the last REMARK record, it's:

    st = gemmi.read_structure(path)
    last_remark = st.raw_remarks[-1]

**Reply by lobnaOmar20** [reply]
I have only 1 remark with 57 steps. but this line you wrote will give me the first step only and I want the last one. therefore I will resort to the normal old python reading it as a txt file. 

Thank you very much for the quick reply.
